### ISSUE TITLE
No room and user avatars are loaded

### ISSUE THREAD

**Issue by PureTryOut** [tulir/mautrix-telegram#422]
I've had this for ages: no room or user avatars are shown, although chats work fine and images in chats are send through normally.

This started happening after I switched database locations, but as far as the bridge is concerned it's using the same database as before, just with a different connection string.

**Comment by benneti** [comment]
I have this problem, too. But the avatars where not there from the begining.
Incase this is relevant the telegram account that is puppetet was puppeted before on another server and the avatars worked there.

**Comment by Kimiblock** [comment]
Having this issue. Synapse complains that:

```
2024-04-28 21:24:11,042 - synapse.handlers.profile - 336 - WARNING - PUT-2817 - Forbidding avatar change to mxc://moechat.kimiblock.top/3e8753a6aff0516adf1b7f6466b5ff6a5418b5c21772634165963915264: avatar not on server
```

**Comment by maltee1** [comment]
Did you delete your local media without using the api for it?

**Comment by Kimiblock** [comment]
No. But the media was imported from MMR and cannot be found. What should I do to force a re-download?

**Comment by Kimiblock** [comment]
So basically what I did to solve the problem is to wipe `avatar_set`, `avatar_url` in any table and deleted `telegram_file`. Seems working again.
